src/test/java/uk/gov/hmcts/sscs/personalisation/PersonalisationTest.java

```diff
@@ -58,6 +58,7 @@ public void setup() {
         when(config.getManageEmailsLink()).thenReturn(Link.builder().linkUrl("http://link.com/manage-email-notifications/mac").build());
         when(config.getClaimingExpensesLink()).thenReturn(Link.builder().linkUrl("http://link.com/progress/appeal_id/expenses").build());
         when(config.getHearingInfoLink()).thenReturn(Link.builder().linkUrl("http://link.com/progress/appeal_id/abouthearing").build());
+        when(config.isJobSchedulerEnabled()).thenReturn(true);
```

Comment:
I see here you are expecting the value to be true. However, in the project that variable is always false including in the /infrastructure.